quickstart/armv7a9-zynq7000-zedboard.md

```diff
@@ -106,7 +113,7 @@ copy usb0 phoenix-armv7a9-zynq7000-zedboard.disk flash0 0x0 0x0
 
 <img src="_images/zynq7000-plo-copy.png" width="600px">
 
-### Booting Phoenix-RTOS from internal Flash
+### Booting Phoenix-RTOS from internal flash
```

Comment:
It is not an internal flash memory.
Change convention to NOR flash memory in the whole file.

Reply:
> It is not an internal flash memory. Change convention to NOR flash memory in the whole file.

Thanks, corrected.